src/routes/index.tsx

```diff
@@ -24,6 +25,69 @@ import {
   LineChartOutlined,
 } from '@ant-design/icons'
 
+export const getPages = () => {
```

Comment:
```suggestion
export const usePages = () => {
```
Every function that calls react hook is a hook